irohad/main/server_runner.cpp

```diff
@@ -39,36 +47,68 @@ iroha::expected::Result<int, std::string> ServerRunner::run() {
   builder.SetMaxReceiveMessageSize(INT_MAX);
   builder.SetMaxSendMessageSize(INT_MAX);
 
-  serverInstance_ = builder.BuildAndStart();
-  serverInstanceCV_.notify_one();
+  server_instance_ = builder.BuildAndStart();
+  server_instance_cv_.notify_one();
 
   if (selected_port == 0) {
     return iroha::expected::makeError(
-        (boost::format(kPortBindError) % serverAddress_).str());
+        (boost::format(kPortBindError) % server_address_).str());
   }
 
   return iroha::expected::makeValue(selected_port);
 }
 
 void ServerRunner::waitForServersReady() {
-  std::unique_lock<std::mutex> lock(waitForServer_);
-  while (not serverInstance_) {
-    serverInstanceCV_.wait(lock);
+  std::unique_lock<std::mutex> lock(wait_for_server_);
+  while (not server_instance_) {
+    server_instance_cv_.wait(lock);
+  }
+}
+
+iroha::expected::Result<std::shared_ptr<grpc::ServerCredentials>, std::string>
+ServerRunner::createSecureCredentials() {
```

Comment:
is this function supposed to ever return *error*? In its body i can see no such option. If so, why wrapping returned object into `Result`?